<launch>
	<arg name="models_dir" default="/opt/strands/data/object_recognition/models" />
	<arg name="recognizer_structure_sift" default="/opt/strands/data/object_recognition/recognizer_structure" />
	<arg name="topic" default="/camera/depth_registered/points" />
	<arg name="training_dir_sift" default="/media/Data/datasets/willow_dataset/sift_trained/" />
	<arg name="gt_or_output_dir" default="" />
	<arg name="visualize_output" default="false" />
	<arg name="icp_iterations" default="3" />
	<arg name="mv_keypoints" default="0" />
	<arg name="opt_type" default="0" />
	<arg name="chop_z" default="2.5" />
	<arg name="training_dir_ourcvfh" default="" />
	<arg name="do_sift" default="true" />
	<arg name="do_ourcvfh" default="false" />
	<arg name="ignore_color" default="false" />
	<arg name="go_3d" default="true" />
	
  <!-- Rumblepad control -->
	<!-- <node pkg="scitos_teleop" type="rumble_control" name="teleop_joystick" output="screen" /> -->
  
  <!--  Multiview recognizer -->
  <node pkg="multiview_object_recognizer" type="multiview_object_recognizer_node" name="multiview_object_recognizer_node" output="screen" >
       <rosparam file="$(find multiview_object_recognizer)/cfg/params_dump.yaml" command="load"/>
  </node>

	
	<!-- recognition service service -->
	<!-- <node pkg="singleview_object_recognizer" type="recognition_service" name="recognition_service" output="screen" >
		<param name="models_dir" value="$(arg models_dir)" type="string"/>
		<param name="training_dir_sift" value="$(arg training_dir_sift)" type="string"/>
		<param name="recognizer_structure_sift" value="$(arg recognizer_structure_sift)" type="string"/>
		<param name="icp_iterations" value="$(arg icp_iterations)" type="int"/>
		<param name="do_sift" value="$(arg do_sift)" type="bool"/>
		<param name="do_ourcvfh" value="$(arg do_ourcvfh)" type="bool"/>
		<param name="chop_z" value="$(arg chop_z)" type="double"/>
		<param name="ignore_color" value="$(arg ignore_color)" type="bool"/>
	</node> -->
</launch> 
